src/main/kotlin/com/terraformation/backend/documentproducer/db/manifest/ManifestImporter.kt

```diff
@@ -289,14 +203,15 @@ class ManifestImporter(
               renderHeading = !csvVariable.isNonNumberedSection))
 
       if (csvVariable.defaultSectionText != null) {
-        val regex = Regex("(.*?)(?:\\{\\{([^}]+)}}|\$)", RegexOption.DOT_MATCHES_ALL)
+        val regex =
+            Regex("(.*?)(?:\\{\\{(?:[^{]+-\\s*)?([0-9]+)}}|\$)", RegexOption.DOT_MATCHES_ALL)
```

Comment:
Should the bit before the hyphen be `[^}]+` instead of `[^{]+`? The way it is now would stop at the beginning of the next variable reference rather than the end of the current one.

Also, maybe `*` instead of `+` so a reference like `{- 1234}` works (right now it would fail because there needs to be at least one character before the hyphen).